test/deprecated/complete.chpl

```diff
@@ -0,0 +1,12 @@
+class C {
+    var x = 1;
+
+    proc doSomething() {
+        writeln(x);
+    }
+
+    proc init() {
+        this.complete();
+        this.doSomething();
+    }
+}
```

Comment:
From a black-box testing perspective, it seems like we should have a case that calls just `complete();` as well to make sure it gets a deprecation warning.  If, from a white-box development perspective, you are confident that that case is covered, there's no need to add it for my sake.